tests/p_dhcp/bind_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+# Author: Petr Menšík <pemensik@redhat.com>
+
+t_Log "Running $0 - dhcp-client: can upgrade client"
+
+dnf upgrade --assumeno 'bind*' 'dhcp*'
```

Comment:
Can you either conditionalize this to CentOS versions >= 8 or switch to calling `yum` ? We need to make sure that this runs (or skips) appropriately on CentOS Linux 7